### PR TITLE
Scopus retrieve aritcle

### PR DESCRIPTION
**Pull request by pontusthome**
Added retrieve article in scopus chain

### CODE REVIEW

File: app/chain/scopus/scopus.resource.js [hunk 1]
```diff
@@ -76,3 +77,19 @@ exports.retrieveAbstract = function(title, params) {
         .then(responseHandler.parseGet)
         .catch(errorHandler.throwResourceError);
 };
+
```
Comment (letterix) [inline app/chain/scopus/scopus.resource.js:81]: newline

Comment (pontusthome) [inline app/chain/scopus/scopus.resource.js:81]: @letterix All set sir

File: app/chain/alchemy/alchemy.routes.js [hunk 2]
```diff
@@ -9,8 +9,8 @@ var alchemyController = require('./alchemy.controller');
 
 module.exports = function(routes) {
 
-    // This just a hard coded test
     routes.get('/:url', function(request, response) {
+		console.log("alchemy");
```
Comment (letterix) [inline app/chain/alchemy/alchemy.routes.js]: remove

File: app/chain/scopus/scopus.controller.js [hunk 3]
```diff
@@ -50,3 +54,150 @@ exports.retrieveAbstract = function(title) {
...
+            }
+            return scopusController.retrieveIssn(issn)
+                .then(getIssnData);
+        })
+}
+
+function getIssnData(issnBody) {
+    if (issnBody === undefined) {
+        var result = {}
+    }
```
Comment (letterix) [inline app/chain/scopus/scopus.controller.js]: else on same line as ending if

File: app/chain/scopus/scopus.controller.js [hunk 4]
```diff
@@ -50,3 +54,150 @@ exports.retrieveAbstract = function(title) {
...
+    })
+}
+
+function extractInfo(jsonBody) {
+    return Promise.filter(jsonBody['search-results'].entry, filterNoAffiliations)
+        .map(function(entry) {
+            var issn = entry['prism:eIssn'];
+            if (issn === undefined) {
+                issn = entry['prism:issn'];
+            }
```
Comment (letterix) [inline app/chain/scopus/scopus.controller.js:171]: correct me if i'm wrong, but should be a newline after if's?

Comment (pontusthome) [inline app/chain/scopus/scopus.controller.js:171]: Added a newline after each if statement

File: app/chain/scopus/scopus.controller.js [hunk 5]
```diff
@@ -50,3 +54,151 @@ exports.retrieveAbstract = function(title) {
...
+    } else {
+        var res = issnBody['serial-metadata-response']['entry'][0];
+        var result = {
+            IPP: res['IPPList']['IPP'][0]['$'],
+            SJR: res['SJRList']['SJR'][0]['$'],
+            SNIP: res['SNIPList']['SNIP'][0]['$']
+        };
+    }
+
+    return new Promise(function(resolve, reject) {
```
Comment (letterix) [inline app/chain/scopus/scopus.controller.js:160]: not like this

File: app/chain/scopus/scopus.controller.js [hunk 6]
```diff
@@ -50,3 +54,151 @@ exports.retrieveAbstract = function(title) {
...
+        return resolve(result);
+    })
+}
+
+function extractInfo(jsonBody) {
+    return Promise.filter(jsonBody['search-results'].entry, filterNoAffiliations)
```
Comment (letterix) [inline app/chain/scopus/scopus.controller.js:166]: like this

**Pull request closed without merging**